_dashboards/visualize/vega.md

```diff
@@ -190,3 +194,122 @@ The following is an example Vega specification with `Demo US Cluster` as the spe
 }
 ```
 {% include copy-curl.html %}
+
+## Creating Vega visualizations with OpenSearch Dashboards
```

Comment:
```suggestion
## Creating Vega visualizations using multiple data sources with OpenSearch Dashboards
```

This title should be revised. These steps reference creating a Vega visualization with MDS enabled. Vega can still function without MDS and steps for these have not been added.